lib/index.js

```diff
@@ -170,14 +181,45 @@ export function createUnifiedLanguageServer({
   const documents = new TextDocuments(TextDocument)
   /** @type {Set<string>} */
   const workspaces = new Set()
+  /** @type {UnifiedLanguageServerSettings} */
+  const defaultSettings = {requireConfig: false}
+  let globalSettings = defaultSettings
+  /** @type {Map<string, Promise<UnifiedLanguageServerSettings>>} */
+  const documentSettings = new Map()
   let hasWorkspaceFolderCapability = false
+  let hasConfigurationCapability = false
+
+  /**
+   * @param {string} scopeUri
```

Comment:
Later `textDocument.uri`s are passed. `scopeUri` sounds more like a (workspace?) “folder” to me than a file path. Perhaps a different name?

Reply:
I matched the named variable passed into `scopeUri`. The [VSCode extension [guide](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#using-configuration-settings-in-the-server) uses `resource`. Either is fine with me.

Reply:
I’m going to keep the name `scopeUri`. This is consistent with the [terminology used by LSP](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#configurationItem).